src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -534,6 +553,25 @@
         ]
       }
     },
+    "ciTargetDotNetVersion": {
+      "type": "generated",
+      "generator": "switch",
+      "replaces": "$ciTargetDotNetVersion$",
```

Comment:
Use 6.0 and 7.0 as target dotnet version. On the off-chance that they ship a 6.1 or 7.1 it'll make it easier to update the template